src/main/java/seedu/address/storage/JsonAdaptedUserImage.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.storage;
+
+import java.util.logging.Logger;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.property.Region;
+import seedu.address.model.userimage.FilePath;
+import seedu.address.model.userimage.UserImage;
+
+
+/**
+ * Constructs a {@code JsonAdapterUserImage} with the given UserImage details.
+ */
```

Comment:
Same as above

Reply:
Thanks for spotting it. It has been removed